src/gui/src/mainWindow.cpp

```diff
@@ -513,6 +514,10 @@ void MainWindow::createActions()
 
   find_ = new QAction("Find", this);
   find_->setShortcut(QString("Ctrl+F"));
+
+  goto_position_ = new QAction("Goto position", this);
```

Comment:
should this be "Go to position"?

Reply:
In klayout is written the same way, I can separate it if you think it is better.

Reply:
"Go to" is better as it is an English phrase.  Goto is just a programming keyword.